### ISSUE TITLE
[Feature Request] Add a Serach to Edit/Context menu

### ISSUE THREAD

**Issue by Mannshoch** [sqlitebrowser/sqlitebrowser#1338]
### Add a Search entry to "Edit"... 
which is able to search within the whol Database not only in some tables
![grafik](https://user-images.githubusercontent.com/7795891/37094679-e6062056-2213-11e8-8ba9-a97fb0241ed6.png)

### Add a "Search" into Context-menu... 
to be able to search a cell-content in the whole database.
![grafik](https://user-images.githubusercontent.com/7795891/37094898-aaffcd30-2214-11e8-9749-1b25d592968b.png)

![grafik](https://user-images.githubusercontent.com/7795891/37094842-8054124e-2214-11e8-9b50-fff92a90b48c.png)

**Comment by MKleusberg** [comment]
What version are you using, @Mannshoch? At least in the nightly build (download link on the main page of our repository) you'll already get your second feature request :smile: 

![screenshot_20180307_192151](https://user-images.githubusercontent.com/3153504/37110327-e108f258-223c-11e8-9c9d-a20ce6d8e066.png)

**Comment by mgrojo** [comment]
I've understood, for searching in all the table cells, not only in a column.

But in fact the third request is already in the development version, but only if you switch to the XML or JSON modes, since it is only implemented for the Scintilla editor:

![imagen](https://user-images.githubusercontent.com/13812910/37114862-d4f42322-2249-11e8-8b24-e78ceb4ec968.png)

It would make sense to implement the same dialog for the general text editor. But the trick of switching to XML/JSON mode works for any text, you only have to ignore the errors about invalid JSON/XML.

**Comment by mgrojo** [comment]
@Mannshoch Do you mean for the first request the same than  #147?

**Comment by MKleusberg** [comment]
@mgrojo Yes, that was the one I meant. I only remembered it to be a very old issue :wink: And yeah, I missed the "in the whole database" bit. Only thought that would apply to the first suggestion :smile:

**Comment by Llaves** [comment]
@MKleusberg - the "select as filter" does not appear in Windows version 3.10.1, downloaded this morning (June 3, 2018)


![dbview](https://user-images.githubusercontent.com/6778818/40887915-1e07f61c-670d-11e8-9208-4f5b10792f62.JPG)

**Comment by justinclift** [comment]
@Llaves Ahhh, yeah.  It's recently added code, so hasn't yet made it's way into our stable releases.

It **is** in our nightly builds though, which you can download here:

&nbsp; &nbsp; https://nightlies.sqlitebrowser.org/latest/

Sorry for the mass of files there.  We're trying out some new installer versions, so there's an overabundance of options.  Just pick one of the .msi files, and you should be good. :smile:
